doc/support.rst

```diff
@@ -35,13 +35,13 @@ for example with `venv <https://docs.python.org/3/library/venv.html>`_:
 
     ~ $ python -m venv ~/envs/signac-flow-dev
     ~ $ source ~/envs/signac-flow-dev/bin/activate
-    (signac-flow-dev) ~ $ pip install six flake8
+    (signac-flow-dev) ~ $ pip install -r requirements-dev.txt
```

Comment:
I'm hesitant about recommending the text file when it has exact version pinnings, instead of `>=`. I understand that those exact requirements should be used for our sanity during tests, but let's avoid forcing contributors to potentially downgrade unless necessary. Perhaps make a separate requirements file for the tests?

Reply:
I agree with that sentiment and I definitely see the issue. Maybe using a [constraint file](https://pip.pypa.io/en/stable/user_guide/#constraints-files) is a way to mitigate this?

Reply:
I don't see how a constraint files solves the problem... I'd consider making a `requirements-test.txt` that pins `==` exact versions and a `requirements-dev.txt` that uses `>=` pinnings.